spec/ADAS/ADAS.vspec

```diff
@@ -10,18 +10,14 @@
 #
 # All Advanced Driver Assist System signals
 #
-
-#  ABS
-#  CruiseControl
-#    TargetSpeed
-#	Engaged
-#	DynamicDistance
-#  TorqueVectoring
-#  BrakeAssist
-#  LaneAssist
-#  ParkAssist
-#  AntiSkid (Better name)
-#  TerrainMode
+# Three signals are commonly used to describe status of a system
+#
+# For systems that most of the time are in standby in the background (like ABS) two signals are often used:
+# IsEnabled indicates that the system has been turned on and is ready to take actions
+# IsEngaged indicates that the system is currently taking action (e.g. regulating brake pressure)
+#
+# For systems that are taking action over an longer period of time (like Cruise Control) one signal is used:
+# IsActive - indicates that system is continuously taking action (e.g. regulating speed and brakes)
```

Comment:
One can also see the different states from another perspective:

ABS is both a sensor (enabled, not engaged) and an actor / actuator (enabled, engaged)
Cruise Control is only an actor / actuator (enabled, engaged)

Therefore it could make sense to have a single term for describing the "acting" state.

Reply:
Will try to rephrase it a bit., but possibly not use the terms sensor/actuator as above as it might be a bit confused with the VSS definition of sensor/actuator signals.